benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json

```diff
@@ -5,7 +5,12 @@
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "summaryConfidentialityRequired", "FieldShowCondition": "appeal.appellant.confidentialityRequired!=\"\"", "TabFieldDisplayOrder": 32},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "summary_sscs", "TabLabel": "Summary", "TabDisplayOrder": 1, "CaseFieldID": "isFqpmRequired", "FieldShowCondition": "isFqpmRequired!=\"\"", "TabFieldDisplayOrder": 39},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "isFqpmRequired", "TabFieldDisplayOrder": 38, "FieldShowCondition": "isFqpmRequired!=\"\""},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "caseName", "TabFieldDisplayOrder": 39},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "caseNameHmctsInternal", "TabFieldDisplayOrder": 39},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "caseNameHmctsRestricted", "TabFieldDisplayOrder": 40},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "caseNamePublic", "TabFieldDisplayOrder": 41},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "ogdType", "TabFieldDisplayOrder": 42},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "caseManagementCategory", "TabFieldDisplayOrder": 43},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "caseAccessCategory", "TabFieldDisplayOrder": 44},
```

Comment:
I'm not sure we need to display all of these. Worth confirming with Femi I think